src/devices_models/devices/AC_branches.jl

```diff
@@ -177,22 +121,28 @@ function branch_rate_constraints!(
             variable_name(FLOW_REACTIVE_POWER_TO_FROM, B),
         ),
     )
-
     return
-
 end
 
 #################################### Flow Limits Constraints ##################################################
 function branch_flow_constraints!(
     psi_container::PSIContainer,
     devices::IS.FlattenIteratorWrapper{PSY.MonitoredLine},
     model::DeviceModel{PSY.MonitoredLine, FlowMonitoredLine},
-    ::Union{Type{PM.DCPPowerModel}, Type{StandardPTDFModel}},
+    ::Type{T},
     feedforward::Union{Nothing, AbstractAffectFeedForward},
-)
+) where {T <: PM.DCPPowerModel}
     flow_range_data = Vector{PSI.DeviceRange}(undef, length(devices))
     for (ix, d) in enumerate(devices)
-        minmax = (min = PSY.get_flowlimits(d).to_from, max = PSY.get_flowlimits(d).from_to)
+        if PSY.get_flowlimits(d).to_from != PSY.get_flowlimits(d).from_to
+            @info("Flow limits in Line $(PSY.get_name(d)) aren't equal. The minimum will be used in formulation $(T)")
+        end
+        limit = min(
```

Comment:
This could create some confusing behavior for the user. For example, if they adjust their data because of a binding flow limit, but only adjust the data in `get_flowlimits(d)`,  they may get a new binding flow limit from the `rate` value. This sort of auto-detection is a bit scary. Could we just stick to flow limits here? If not, we should consider adding an interface in a future PR like `set_line_limits` or something that sets rate and flow_limits consistently.

Reply:
The thing here is that we are creating way too many bounds/constraints that have the same value. In some cases is useful to have both depending on the solution algorithm for instance OSQP takes fewer iterations to solve when bounds and constraints are combined. But for larger systems, we could end up doubling the branch flow limits and making it worse. 

Also, I don't know if it makes sense to impose a flow limit larger than the rate that would be a data error. I suggest we leave like this and revisit when resolving the PSIMA story about selective enforcing of flow limits.